packages/auth/__tests__/providers/cognito/utils/oauth/attemptCompleteOAuthFlow.test.ts

```diff
@@ -108,8 +99,8 @@ describe('attemptCompleteOAuthFlow', () => {
 		expect(mockCompleteOAuthFlow).not.toHaveBeenCalled();
 	});
 
-	it('sets inflight oauth promise and invokes `completeOAuthFlow` to complete an inflight oauth process', async () => {
-		(oAuthStore.loadOAuthInFlight as jest.Mock).mockResolvedValueOnce(true);
+	it('invokes `completeOAuthFlow` to complete an inflight oauth process', async () => {
+		(oAuthStore.loadOAuthInFlight as jest.Mock).mockResolvedValueOnce(Promise.resolve(true));
```

Comment:
nit: As we use `mockResolvedValueOnce` here, probably don't need the `Promise.resolve()` right?

```suggestion
		(oAuthStore.loadOAuthInFlight as jest.Mock).mockResolvedValueOnce(true);
```